### ISSUE TITLE
[FEATURE] pdfs zusammenlegen

### ISSUE THREAD

**Issue by stm9x9** [jlawyerorg/j-lawyer-org#1513]
![image](https://user-images.githubusercontent.com/29924198/147791181-de944729-0b2d-4db1-9bea-bb13f3306f62.png)

Aus Advoware inspiriert: pdfs in der Liste anklicken und dann zusammenführen. Erst eine Liste der angewählten Dateien zeigen (ob als Text oder besser mit Vorschau der ersten Seite), dort eine Reihenfolge auswählen und dann sollte (serverseitig) ein neues pdf erzeugt werden mit Abfrage, wie es heißen soll. 

Sowas braucht ich zb regelmäßig, wenn ich Unterlagen für einen Beratungshilfeantrag zusammenstelle.

Ggf könnte die Funktion auch "als pdf zusammenlegen" heißen, wenn es möglich sein sollte, dabei auch nicht pdfs zu berücksichtigen. Zb bekomme ich oft Dokumente von Mandanten als Einzelfotographien in jpg. Die muß ich dann immer manuell in pdfs umwandeln und dann zusammenfügen.

**Comment by j-dimension** [comment]
siehe auch #104

**Comment by Thorsten-usr** [comment]
Ja, habe auch das Problem, Bedürfnis meine Deckblätter für Anlagen mit den jeweiligen Anlagen zusammenzuführen. Ich benutze dafür momentan MasterPdf in der Freeversion. Das Programm habe ich als Pdf-Viewer in J-Lawyer hinterlegt. Schön wäre, wenn man mehrere Programme zur Bearbeitung von Pdf-Dateien hinterlegen könnte.

**Comment by j-dimension** [comment]
"Schön wäre, wenn man mehrere Programme zur Bearbeitung von Pdf-Dateien hinterlegen könnte."

Dafür bitte ein separates Ticket. Danke

**Comment by iradraconis** [comment]
> "Schön wäre, wenn man mehrere Programme zur Bearbeitung von Pdf-Dateien hinterlegen könnte."
> 
> Dafür bitte ein separates Ticket. Danke

#1345 habe ich mir derselben Intention aufgemacht.

**Comment by j-dimension** [comment]
siehe auch #1794

**Comment by j-dimension** [comment]
Mittlerweile verfügbar.
